src/blocks/PlainTextBlock.tsx

```diff
@@ -1,54 +1,57 @@
 import styled from '@emotion/styled'
-import { colors } from '@hedviginsurance/brand'
 import React from 'react'
-import { ContentWrapper, SectionWrapper } from '../components/blockHelpers'
-import { BaseBlockProps, MarkdownHtmlComponent } from './BaseBlockProps'
+import {
+  ContentWrapper,
+  MOBILE_BP_UP,
+  SectionWrapper,
+} from '../components/blockHelpers'
+import {
+  BrandPivotBaseBlockProps,
+  MarkdownHtmlComponent,
+} from './BaseBlockProps'
 
-const PlainTextSectionWrapper = styled(SectionWrapper)({
-  paddingBottom: '0 !important',
-})
+type ParagraphFontSizes = 'sm' | 'md' | 'lg' | 'xl'
 
-const TitleWrapper = styled('div')({
-  paddingBottom: '2.5rem',
-})
+const paragraphSizeMap = {
```

Comment:
Just an idea - what about specifying this as `Record<ParagraphFontSize, string>`?

Reply:
Nice!